app/Http/Controllers/Backend/CategoryController.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests\Backend\CategoryRequest;
+use App\Http\Controllers\Controller;
+use App\Models\Backend\Category;
+use File;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the products.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $cates = Category::paginate(10);
+        $parent = Category::select('id', 'cate_name', 'parent_id')->get()->toArray();
+        return view('backend.categories.index', compact('cates', 'parent'));
```

Comment:
[rank C] In there, we need declare everything more clearly. Please consider about my code:
```
$data["cats"] = Category::paginate(ITEM_PER_PAGE);
$data["parent"] = Category::select('id', 'cate_name', 'parent_id')->get()->toArray();
return view('backend.categories.index')->with($data);
```